### PR TITLE
specify application deadline date, correct typo

### PR DESCRIPTION
**Pull request by BartBucknill**
Specify final date for this round of London applications as July 31st; more specific than previous 'end of July'.

#437

### CODE REVIEW

**Comment by jsms90** [comment]
@sofer @rebeccaradding Was I correct in saying 31st July?

File: _harp/apply/_london.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <h2 align='center'>SELECTION PROCESS</h2>
 
-We will review applications and offer interviews at the end of July, including a guranteed interview to all candidates who have completed the course prerequisites. Interviews will be conducted in-person (or via Google hangouts) in early August. We will select 16 people for the course beginning in October.
+From 31st July we will review applications and offer interviews, including a guaranteed interview to all candidates who have completed the course prerequisites. Interviews will be conducted in-person (or via Google hangouts) in early August. We will select 16 people for the course beginning in October.
```
Comment (jsms90) [inline _harp/apply/_london.md]: Two different ways of writing dates in the same file:
> 31st July

> 18 February

We should pick one style and stick to it.

**Comment by BartBucknill** [comment]
@jsms90 Changed date to '31 July' for consistency in response to change request.

**Comment by rebeccaradding** [comment]
@BartBucknill I'm happy to merge this, but you or @jsms90 will need to deploy the changes...

**Comment by jsms90** [comment]
@Jen-Harris @bowssy88 Remind me to walk you through the process, since we were talking about the website coming under your biz dev task group remit :+1:

**Comment by BartBucknill** [comment]
@jsms90 just had someone on the phone asking for the specific deadline date. Is it possible for us to deploy this to the live website yet?

**Comment by jsms90** [comment]
@BartBucknill Good point. Had someone ask me too.

Deployed :+1:

**Pull request merged**